src/utils.js

```diff
@@ -348,28 +348,28 @@ export function parseSrcset(input) {
   }
 }
 
-export function parseSrc(input) {
```

Comment:
It was by design to reuse in some packages, so we should export it

Reply:
Ok, I'm rolling it back